packages/css/src/unordered-list/unordered-list.scss

```diff
@@ -29,7 +38,24 @@
   @include reset;
 }
 
-.amsterdam-unordered-list__item {
+.amsterdam-unordered-list > .amsterdam-unordered-list__item {
+  padding-inline: var(--amsterdam-unordered-list-unordered-list-item-padding-inline);
```

Comment:
This padding should only apply to list items in nested lists, right? I think it applies to all list items now

Reply:
Let’s fix all padding when we update the list gap in DES-240.